lc3/build.sh

```diff
@@ -22,15 +24,13 @@ LAST_REVISION="${3:-}"
 initialise "${REVISION}" "${OUTPUT}" "${LAST_REVISION}"
 
 PREFIX=$(pwd)/prefix
-DIR=$(pwd)/C-LC3-Compiler
-BUILD=${DIR}/build
+DIR=$(pwd)/lc3-compiler
+BUILD=${DIR}/target/release
 
-git clone --recurse-submodules --depth 1 -b "${BRANCH}" "${URL}" "${DIR}"
+git clone --depth 1 -b "${BRANCH}" "${URL}" "${DIR}"
 
-mkdir "${BUILD}"
-cd "${BUILD}"
-cmake ..
-make
+cd $DIR
+cargo build --release
```

Comment:
Is it statically built? will the binary have any dependencies on the rust runtime?